app/src/main/java/org/mozilla/fenix/tabstray/browser/NormalBrowserTrayList.kt

```diff
@@ -23,17 +23,17 @@ import java.util.concurrent.TimeUnit
  */
 const val DEFAULT_ACTIVE_DAYS = 4L
 
+/**
+ * The maximum time from when a tab was created or accessed until it is considered "inactive".
+ */
+val maxActiveTime = TimeUnit.DAYS.toMillis(DEFAULT_ACTIVE_DAYS)
```

Comment:
I just wasn't sure where to refactor/relocate this to

Reply:
For now, this is fine. I think we need to eventually move it somewhere else when the next set of settings land.